karpenter-1.0.yaml

```diff
@@ -1,10 +1,14 @@
 package:
-  name: karpenter
+  name: karpenter-1.0
   version: 1.0.6
   epoch: 0
```

Comment:
Still needs an epoch bump to make sure this one with provides wins out right?

Reply:
I am not so sure about this, since we renamed the package, do we still need epoch bump?

Reply:
No epoch is needed here